src/libraries/System.Private.Runtime.InteropServices.JavaScript/src/System/Runtime/InteropServices/JavaScript/Marshallers/BaseMarshaller.cs

```diff
@@ -0,0 +1,16 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+namespace System.Runtime.InteropServices.JavaScript.Private
+{
+    internal abstract class BaseMarshaller<T> : ICustomJavaScriptMarshaller
```

Comment:
While we aren't going to copy much about what they actually did, our names should be consistent with existing names in the BCL and runtime https://docs.microsoft.com/en-us/dotnet/api/system.runtime.interopservices.icustommarshaler?view=net-6.0